packages/cli/src/util/arg-common.ts

```diff
@@ -28,6 +25,10 @@ const ARG_COMMON = {
   '--api': String,
 
   '--cwd': String,
+
+  // Deprecated
+  '--platform-version': Number,
+  '-V': '--platform-version',
```

Comment:
Removed these from help outputs. This isn't used anywhere that I can find. I left the reference here so that existing scripts wouldn't complain.

Reply:
Also see https://github.com/vercel/vercel/pull/9807.